### PR TITLE
Add ability to add downvotes (Missing reaction type)

### PR DESCRIPTION
**Pull request by paulxuca**
https://avara.height.app/T-4999

### CODE REVIEW

File: packages/react/src/publication/adapters/ReactionPresenter.ts [hunk 1]
```diff
@@ -39,46 +50,37 @@ export class ReactionPresenter
...
-        }
-        break;
-
-      case ReactionType.DOWNVOTE:
-        throw new Error('Downvotes support not implemented');
-      default:
-        throw new Error('Unknown reaction type');
-    }
+    this.publicationCacheManager.update(request.publicationId, (current) => {
+      const removedStatKey = current.reaction
```
Comment (krzysu) [inline packages/react/src/publication/adapters/ReactionPresenter.ts:54]: what is `removedStatKey` for?

**Comment by krzysu** [comment]
could you also add tests for downvote case and for both upvote and downvote, I remember when I was testing them both manually with API the results were confusing, sth like downvote was removing upvote, but I don't remember the details now.

File: packages/react/src/publication/adapters/ReactionPresenter.ts (hunk 1 above)
Comment (paulxuca) [reply packages/react/src/publication/adapters/ReactionPresenter.ts:54]: If we add a downvote to a publication that already has an upvote, we have to remove 1 from the stat value that the active profile previously had -- `removedStatKey` is that key in `PublicationStats` (`totalUpvotes` | `totalDownvotes`)

**Review by krzysu — APPROVED** [review]
👍

**Pull request merged**